docs/src/index.md

```diff
@@ -4,9 +4,12 @@
 ```@meta
 CurrentModule = TSVD
 DocTestSetup = quote
-    using MatrixDepot, TSVD
+    using MatrixDepot, TSVD, Random
```

Comment:
You will need https://github.com/fredrikekre/Literate.jl/blob/d43e75a69e49a7def6f9059d14f83bd442f00d7b/docs/make.jl#L1-L4 at the top of docs/make.jl since this quote will be `eval`d by Documenter after changing directories.